src/models/TableModel/impl/InMemoryTable.ts

```diff
@@ -10,10 +11,25 @@ import { AttributeDeclaration } from '../../../utils/cx/Cx2/CoreAspects/Attribut
 
 export const createTable = (id: IdType): Table => ({
   id,
-  columns: new Map<AttributeName, ValueTypeName>(),
+  columns: new Map<AttributeName, Column>(),
   rows: new Map<IdType, Record<AttributeName, ValueType>>(),
 })
 
+// a particular row may not have a value associated with it and a column may not define the default value
+// in such cases fallback to the model defined default value
+export const valueTypeDefaults: Record<ValueTypeName, ValueType> = {
```

Comment:
Remove this.  Do not populate default values if they are missing from rows unless the column definition specifies it.